content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -26,3 +51,37 @@ version. [Learn more about availability during upgrades here](../system-overview
     "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
+
+### Replicated Command Line Interface
+
+1. Login to a command line terminal of the Terraform Enterprise host machine.
+1. Execute the command below and take note on the location from the property `AirgapPackagePath`:
+   
+    ```
+    replicatedctl params export | grep AirgapPackagePath
+    ```
+
+1. Upload the desired airgap packages into the location of `AirgapPackagePath` from the previous step on the Terraform Enterprise host machine.
```

Comment:
@sudomateo am I correct in recalling that the filename must be different than the previous airgap package?

Reply:
I don't believe they _need_ to be different, but generally people keep the name that's given upon download which is `APPLICATION - SEQUENCE.airgap`. For example, `Terraform Enterprise - 520.airgap`.